docs/articles/expensify-classic/billing-and-subscriptions/Free-Trial.md

```diff
@@ -1,5 +1,44 @@
 ---
 title: Free Trial
-description: Free Trial
+description: Learn more about your free trial with Expensify.
 ---
-## Resource Coming Soon!
+
+# Overview
+New customers can take advantage of a seven-day Free Trial on a group Workspace. This trial period allows you to fully explore Expensify's features and capabilities before deciding on a subscription. 
+During the trial, your organization will have complete access to all the features and functionality offered by the Collect or Control workspace plan. This post provides a step-by-step guide on how to begin, oversee, and successfully conclude your organization's Expensify Free Trial.
+
+# How to start a Free Trial 
+1. Sign up for a new Expensify account at expensify.com. 
+2. After you've signed up for a new Expensify account, you will see a task on your Home page card asking if you are using Expensify for your business or as an individual. 
+  a. **Note**: If you select “Individual”, Expensify is free for individuals for up to 25 SmartScans per month. Selecting Individual will **not** start a Free Trial. More details on individual subscriptions can be found here.
+3. Select the Business option.
+4. Select which Expensify features you'd like to set up for your organization.
+5. Congratulations, your seven-day Free Trial has started!
+
+Once you've made these selections, we'll automatically enroll you in a Free Trial and create a Group Workspace, which will trigger new tasks on your Home page to walk you through how to configure Expensify for your organization. If you have any questions about a specific task or need assistance setting up your company, you can speak with your designated Setup Specialist by clicking “Support” on the left-hand navigation menu and selecting their name. This will allow you to message your Setup Specialist, and request a call if you need. 
+
+# How to unlock additional Free Trial weeks 
+When you begin a Free Trial, you'll have an initial seven-day period before you need to provide your billing information to continue using Expensify. Luckily, Expensify offers the option to extend your Free Trial by an additional five weeks!
+
+To access these extra free weeks, all you need to do is complete the tasks on your Home page marked with the "Free Week!" banner. Each task completed in this category will automatically add seven more days to your trial. You can easily keep track of the remaining days of your Free Trial by checking the top right-hand corner of your Expensify Home page.
+
+# How to make the most of your Free Trial 
+*Complete all of the "Free Week!" tasks right away. These tasks are crucial for establishing your organization's Workspace, and finishing them will give you a clear idea of how much time you have left in your Free Trial. 
```

Comment:
Are these supposed to be bullet points? I think that would look better. You can do that with `- ` instead.
![image](https://github.com/Expensify/App/assets/26260477/8dedb784-4695-4075-9088-5e0e669ac7bf)

Reply:
They _are_. Ty, updated.